bundles/org.openhab.binding.salus/src/main/resources/OH-INF/i18n/salus.properties

```
@@ -0,0 +1,81 @@
+channel-type.salus.generic-input-channel.label = Generic Input Channel
+channel-type.salus.generic-input-channel.description = This channel type represents a generic input. The channel is write-only and its state is represented as a string.
+channel-type.salus.generic-input-number-channel.label = Generic Number Input Channel
+channel-type.salus.generic-input-number-channel.description = This channel type represents a generic input. The channel is write-only and its state is represented as a numeric.
+channel-type.salus.generic-output-bool-channel.label = Generic Bool Output Channel
+channel-type.salus.generic-output-bool-channel.description = This channel type represents a generic output. The channel is read-only and its state is represented as a boolean.
+channel-type.salus.generic-output-channel.label = Generic Output Channel
+channel-type.salus.generic-output-channel.description = This channel type represents a generic output. The channel is read-only and its state is represented as a string.
+channel-type.salus.generic-output-number-channel.label = Generic Number Output Channel
+channel-type.salus.generic-output-number-channel.description = This channel type represents a generic output. The channel is read-only and its state is represented as a numeric.
+channel-type.salus.it600-expected-temp-channel.label = Expected Temperature
+channel-type.salus.it600-expected-temp-channel.description = Sets the desired temperature in room
+channel-type.salus.it600-temp-channel.label = Temperature
+channel-type.salus.it600-temp-channel.description = Current temperature in room
+channel-type.salus.it600-work-type-channel.label = Work Type
+channel-type.salus.it600-work-type-channel.description = Sets the work type for the device. OFF — device is turned off MANUAL — schedules are turned off, following a manual temperature set, AUTOMATIC — schedules are turned on, following schedule, TEMPORARY_MANUAL — schedules are turned on, following manual temperature until next schedule.
+channel-type.salus.it600-work-type-channel.state.option.OFF = OFF
+channel-type.salus.it600-work-type-channel.state.option.MANUAL = Manual
+channel-type.salus.it600-work-type-channel.state.option.AUTO = Automatic
+channel-type.salus.it600-work-type-channel.state.option.TEMPORARY_MANUAL = Temporary Manual
+channel-type.salus.temperature-input-channel.label = Generic Input Temperature Channel
+channel-type.salus.temperature-input-channel.description = This channel type represents a generic input. The channel is write-only and its state is represented as a temperature (numeric).
+channel-type.salus.temperature-output-channel.label = Generic Output Temperature Channel
+channel-type.salus.temperature-output-channel.description = This channel type represents a generic output. The channel is read-only and its state is represented as a temperature (numeric).
+
+# code i8n
+
+cloud-bridge-handler.initialize.username-pass-not-valid = Username or password is not valid
```

```suggestion
cloud-bridge-handler.initialize.username-pass-not-valid = Username or password is missing
```

Strictly they are not provided. (they are checked for blanks). Only the remote service knows if they are valid.